src/Crowdfunding/N3O.Umbraco.Crowdfunding/Models/Properties/ContentPropertyCriteriaRes.cs

```diff
@@ -0,0 +1,8 @@
+namespace N3O.Umbraco.Crowdfunding.Models;
+
+public class ContentPropertyCriteriaRes {
```

Comment:
Can we call this Configuration instead of Criteria as we use criteria to mean search criteria